### PR TITLE
Fix codeblock format issue and refresh language.

### LINKED ISSUE #2595: Fix codeblock issue and refresh content in README.md
@mmartin24 noticed a problem in a codeblock. I'll take the opportunity to refresh the README.

### PR DESCRIPTION
**Pull request by jhkrug**
Will close #2595

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,60 +27,53 @@ Opinionated platform that runs on Kubernetes to take you from Code to URL in one
...
-[docs.epinio.io](https://docs.epinio.io/) site.
+Documentation is available at [docs.epinio.io](https://docs.epinio.io/).
 
 ### Installation
 
-The basic boilerplate requires a Kubernetes cluster, an Ingress Controller and a Cert Manager as
-detailed in the [documentation](https://docs.epinio.io/installation/install_epinio). Once this is in
-place, and leaving out DNS setup, in the most trivial case the main installation boils down to
+Requires a Kubernetes cluster, an Ingress Controller and a Cert Manager as explained in the [installation](https://docs.epinio.io/installation/install_epinio) documentation.
+Once this is in place, and leaving out DNS setup, an installatio reduces to:
```
Comment (andreas-kupries) [inline README.md:53]:
```suggestion
Once this is in place, and leaving out DNS setup, an installation reduces to:
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
typo

**Review by andreas-kupries — APPROVED** [review]

**Comment by jhkrug** [comment]
Hi @andreas-kupries, I want to check. It says all checks have failed. Should I still merge? It's only the README so not related I assume.

**Comment by andreas-kupries** [comment]
:eyes: ... Looking at the check it fails to log into docker.
Does not have user/password.
Fully unrelated to documentation.
It actually should not have triggered, IMHO, the README is in the list of ignores.
Hm. ... The ignores might be limited to pushes on main, not for PRs :(

Merging it now.

**Pull request merged**